deps/build.jl

```diff
@@ -21,4 +21,10 @@ end
     provides(Homebrew.HB, "hdf5", hdf5, os = :Darwin )
 end
 
+begin
+    hdf5 = library_dependency("libhdf5")
```

Comment:
My guess is that the second call to `library_depenceny` overwrites the previous calls.